.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
```

Comment:
What does labels do? Will this by default auto set issue's label?

Reply:
This sets the label by default. Yes.

Reply:
please remove default. On moment of issue creation we do not know what it will be. We use 'bug' as label to route issue to proper group in release notes.

Reply:
Done!